### ISSUE TITLE
Explain Execution Plan Node Type doesn't include "Parallel" keyword

### ISSUE THREAD

**Issue by yonisade** [dbeaver/dbeaver#14469]
#### System information: 
- Operating system (distribution) and version: Windows 10 64 bit
- DBeaver version: 21.2.4.202111011413
- Java version: 1.8

#### Connection specification:
- Database name and version: PostgreSQL v13.4
- Driver name: PostgreSQL JDBC Driver
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
When generating Explain with "EXPLAIN select ..." the plan appears with "Parallel Seq Scan" as expected, 
but when generating Explain with CTRL+SHIFT+E, the Node Type value appears as "Seq Scan" (while parallel-aware=true on the right side)
It would be more clear and by the standard if DBeaver keeps the "parallel" keyword before "Seq Scan".

**Comment by LonwoLonwo** [comment]
Hello @yonisade 

Sorry, but I'm not sure that I understand you correctly. 
DBeaver just returns an answer from the server for the "EXPLAIN ...." query. DBeaver does not add anything to the result.

![2021-11-03 10_08_05-DBeaver Ultimate 21 2 4 - _postgres_ Script-10](https://user-images.githubusercontent.com/45152336/140021081-16d4671b-28ed-4b5e-b285-0ffd6ef3e4ab.png)

You can check your Explain query and DBeaver's query in the [Query Manager](https://github.com/dbeaver/dbeaver/wiki/Query-Manager).

Can you tell us more about your idea?

**Comment by yonisade** [comment]
Please change your query to "EXPLAIN select i,count(*) from test group by i order by 2 desc"
You're supposed to see "Parallel Seq Scan" under the plan's "Node Type" as opposed to the plan generated by "CTRL+Shift+E"

If you don't see the "Parallel" keyword, you can also try running before: "set force_parallel_mode=on" and try EXPLAIN again.

**Comment by LonwoLonwo** [comment]
Sorry, didn't help. Obviously, the result depends on server settings, which are different on my server.

![2021-11-03 10_33_12-DBeaver Ultimate 21 2 4 - _postgres_ Script-10](https://user-images.githubusercontent.com/45152336/140023290-e9dc6612-9ba5-430e-bae0-50cc20d8c9d0.png)

**Comment by yonisade** [comment]
try:
set max_parallel_workers=4;
set max_parallel_workers_per_gather=2;
set force_parallel_mode=on;
set parallel_setup_cost = 10;
set parallel_tuple_cost = 0.001;

**Comment by Matvey16** [comment]
@yonisade Thank you for the report, I reproduced this bug.

**Comment by Matvey16** [comment]
This happens because the `EXPLAIN (FORMAT XML)` is executed on server side and Postgres returns parallel qualification as separate attribute.

```xml
        <Plan>
         <Node-Type>Seq Scan</Node-Type>
         <Parent-Relationship>Outer</Parent-Relationship>
         <Parallel-Aware>true</Parallel-Aware>
         <Relation-Name>import_time2</Relation-Name>
         <Alias>import_time2</Alias>
         <Startup-Cost>0.00</Startup-Cost>
         <Total-Cost>127019.38</Total-Cost>
         <Plan-Rows>2083038</Plan-Rows>
         <Plan-Width>22</Plan-Width>
        </Plan>
```

Maybe there is a way to parse this

**Comment by LonwoLonwo** [comment]
Fixed in DBeaver 21.3.1

![2021-11-30 10_06_16-DBeaver Ultimate 21 3 0 - _postgres 13 3_Script-81](https://user-images.githubusercontent.com/45152336/144002262-9d003ab0-2025-4551-9da2-18c3312e4c61.png)

**Comment by Matvey16** [comment]
Verified
